cmd/cape/cmd/token.go

```diff
@@ -74,8 +80,24 @@ func token(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not detect your user id, perhaps retry logging in")
 	}
 
+	// Get the un-parsed access token.
+	// (TODO) Optimize token retrieval so we only get auth token once.
+	t, err := authToken()
+	if err != nil {
+		return err
+	}
+	auth := entities.FunctionAuth{Type: entities.AuthenticationTypeAuth0, Token: t}
+	// Check that the signed in user has ownership access to the function before creating
+	// the token for it.
+
+	err = doGet(functionID, url, insecure, auth)
+	if err != nil {
+		return err
```

Comment:
We should use the `UserError` type here (depending on the error). E.g., when I do 

```
$ cape token hMnFfqBnpwkFktLamruDVf
```

I get 

```
Error: unauthorized to create a function token for function
Usage:
  cape token <function_id> [flags]

Flags:
  -e, --expiry int                 optional time to live (in seconds) (default 3600)
      --function-checksum string   optional function checksum
  -h, --help                       help for token
      --owner                      optional owner token (debug logs)

Global Flags:
  -c, --config string   config file (default "$HOME/.config/cape/presets.json")
  -o, --output string   output format (default "plain")
  -v, --verbose         verbose output

Command failed with error.
```

If we noted that this was a user error we would suppress the usage information, which we shouldn't display this this kind of error

Reply:
Yeah, I updated the logic for silencing output, I think the original condition needs to be changed and I did it for the other commands as well.